lesson_16/Exercise_1/exercise_1.py

```diff
@@ -0,0 +1,26 @@
+"""
+Напишіть тест, який перевіряє наявність атрибутів з Manager та Developer у класі TeamLead
+"""
+
+from pytest import fail
+from src1 import *
+
+class TestTeamLeadClassInheritance:
+    def test_manager_atributes_positive(self):
+        # Arrange & Act
+        new_lead = TeamLead()
+        try:
+            assert new_lead.department != None
+        except AttributeError:
+            fail("Manager attribute is missing!")
+
+    def test_developer_atributes_positive(self):
+        # Arrange & Act
```

Comment:
try to make your 2 tests as 1 parameterised, since they have similar structure
and also use __getattr__ and __setattr__ if applicable